src/twisted/web/test/test_http.py

```diff
@@ -2566,10 +2566,14 @@ def process(self):
                 self.finish()
 
         body = b"""-----------------------------738837029596785559389649595
-Content-Disposition: form-data; name="uploadedfile"; filename="test"
+Content-Disposition: form-data; name="uploadedfiles"; filename="testA"
```

Comment:
I think it's ok to extend the existing test.

In an ideal case, we would have a test for a single file, and a separate test for multiple files.

But I think that we should also update the name of this test

```diff
-    def test_multipartFileData(self):
+    def test_multipartFilseData(self):
```

And update the docstring of the test to document that file names are added to the list, in the order as they appear in the request

--------

and very minor thing.

... use a different name, to make sure we don't accidentally sort them by name... 

```suggestion
Content-Disposition: form-data; name="uploadedfiles"; filename="testC"
```

Reply:
Yeah I did wonder about creating a separate test for this scenario but it seems sensible to extend this test.

Fixed as per suggestions.